src/dali/plugin/pair_converter/ccxt.py

```diff
@@ -81,6 +81,7 @@
 }
 
 # Currently supported exchanges
+_ALIAS: str = "Alias"  # Virtual exchange
```

Comment:
Do we need this? I was hoping we could do without fake exchanges.

Reply:
It's admittedly temporary. CCXT needs to know where to look for prices. No class is instantiated for the exchange. This is mostly a flag for CCXT to look up the bar with the `MappedGraph`

Reply:
Can you add a comment to mark this as tech debt to be fixed later?